radar/2024-09-04/cht-sync.md

```diff
@@ -0,0 +1,10 @@
+---
+title:      "CHT Sync"
+ring:       adopt
+quadrant:   methods-and-patterns
+tags:       [data-use]
+---
+
+[CHT Sync](https://github.com/medic/cht-sync) is a integrated solution designed to enable data synchronization between CouchDB and PostgreSQL for the purpose of analytics. The synchronization occurs in real-time, ensuring that the data displayed on dashboards is up-to-date. It combines several technologies to achieve this synchronization (an updated version of [couch2pg](https://github.com/medic/cht-sync/tree/main/couch2pg), CouchDB, PostgreSQL, dbt) and provides an efficient workflow for data processing and visualization.
```

Comment:
"in real-time" is probably overselling it a bit.

Also let's drop couch2pg - it's  an internal detail and not relevant to implementers.

```suggestion
[CHT Sync](https://github.com/medic/cht-sync) is a integrated solution designed to enable data synchronization between CouchDB and PostgreSQL for the purpose of analytics. The synchronization occurs near real-time, ensuring that the data displayed on dashboards is up-to-date. It combines several technologies to achieve this synchronization and provides an efficient workflow for data processing and visualization.
```